content/post/articles/Become-A-Software-Developer.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Become a Software Developer"
+date: 2021-11-30T15:32:19+01:00
+images: ["/Become-A-Software-Developer/shaba-rex.png"]
+categories: ["Software Developer"]
+tags: ["Software Developer","BootCamps","Self-taught Developer"]
+authors: ["Shaba-Rex"]
+---
+
+{{<figure src="/Become-A-Software-Developer/viva the it.jpeg"  alt="Software Engineering is Cool">}}
+
+Software engineering is the process of analyzing user needs and designing, constructing, and testing end-user applications that will satisfy these needs through the use of software programming languages. It is the application of engineering principles to software development.
+
+To be an Application Developer, good news there is not just one path, in this article we will talk about the different paths you have to take to become a software developer and the advantages and disadvantages of certain paths. But in the end it's up to you to decide which path is best for you.
```

Comment:
You talk about being a software developer, yet here you say 
> Application Developer

Do you consider them to be the same thing?

Reply:
Salut frère , j'entends ici par 'Application' toutes les applications pour smartphones, tablettes et ordinateurs.
Je considère le développeur d 'application comme celui là qui crée les  logiciels d'application / les logiciels de système.
Donc oui pour moi il sont pareil.